support-frontend/app/views/contributions.scala.html

```diff
@@ -36,12 +39,17 @@
       @user.publicFields.displayName.map { displayName =>
         displayName: "@displayName",
       }
-    @for(fields <- user.privateFields; firstName <- fields.firstName; lastName <- fields.secondName) {
-        firstName: "@firstName",
-        lastName: "@lastName",
+      @for(fields <- user.privateFields) {
+        @for(firstName <- fields.firstName; lastName <- fields.secondName) {
+          firstName: "@firstName",
+          lastName: "@lastName",
+        }
+        @for(address4 <- fields.address4) {
+          address4: "@address4",
```

Comment:
What is `address4`?

Reply:
It's the Identity API JSON model key name for where the State value is stored.